client/internal/routers/result.go

```diff
@@ -3,7 +3,7 @@ package routers
 import (
 	"sync"
 
-	"github.com/retail-ai-inc/beanq/client/internal/routers/consts"
+	"github.com/retail-ai-inc/client/internal/routers/consts"
 )
```

Comment:
may be it should be `"github.com/retail-ai-inc/beanq/client/internal/routers/consts"`